include/eve/detail/function/simd/common/compress_store_impl.hpp

```diff
@@ -18,6 +18,94 @@
 
 namespace eve::detail
 {
+  // permutation masks ------------------
```

Comment:
shoudln't all of this actually be `consteval`to be sure we don't do any og this at runtime ?